src/Client/ApiKey.php

```diff
@@ -103,4 +107,91 @@ public function createApikey(?string $label = null, ?array $permissions = null):
             throw $this->getExceptionByStatusCode($method, $url, $response);
         }
     }
+
+    /**
+     * Create a new API key for a user.
+     *
+     * @param string $userId Can be user id or email.
+     * @param string $label Visible label on API key overview
+     * @param array $permissions The permissions array can contain specific store id
+     * e.g. btcpay.server.canmanageusers:2KxSpc9V5zDWfUbvgYiZuAfka4wUhGF96F75Ao8y4zHP
+     */
+    public function createApiKeyForUser(
+        string $idOrMail,
+        ?string $label = null,
+        ?array $permissions = null
```

Comment:
What happens if $permissions is NULL or empty array? IIRC permissions cannot be edited once an API key is created, so maybe we should throw an \IllegalArgumentException for empty arrays? It's pointless to create 0 permission keys.

Reply:
Will need to try but afair it would grant all possible permissions? At least for the admin it would be "unrestricted" but not sure in this case.

Reply:
Just checked, it works but the api key will just have no permissions. Not sure for what it can be used then, maybe just for all open accessible stuff, so it is fine to keep it?

Reply:
There is at least the get server info endpoint that requires api key but no specific permission, so it seems to be ok this way.

Reply:
ping @woutersamaey